app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = @user.posts.find(params[:id])
+
+    @like = Like.new(user: @user, post: @post)
+    if @like.save
+      @like.update_likes_counter
+      flash[:success] = 'Like created successfully!'
+
+    else
+      flash[:error] = 'Like not created'
+    end
+    redirect_to user_posts_path(@user.id)
+  end
+end
```

Comment:
- [ ] You did a great job working on the previous reviewer's advice! 💯 However, the like functionality hasn't been implemented yet. According to the requirements:

![image](https://github.com/byiringiroscar/wikipedia-project-with-rails/assets/77069011/a2ad3d23-4413-4044-b155-9c49ba355596)
 
Please consider implementing this!

Reply:
@anagudelogu thanks for the review please check well in all posts likes work very fine check on this route 
get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
please check it again